src/style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-weight: 300;
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.search {
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  width: 100%;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  display: block;
+  margin: 20px 0;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+
+}
+
+.search-input-l {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  background-position: top 25px left 26px;
+  background-size: 19px;
+}
+
+.search-input-s {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  background-position: top 15px left 13px;
+  background-size: 11px;
+}
+
+.search:hover {
+  background: #fff;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  font-weight: normal;
+}
+
+.search-input-l:hover {
+  background-position: top 25px left 26px;
+  background-size: 19 px;
+}
+
+.search-input-s:hover {
+  background-position: top 15px left 13px;
+  background-size: 11px;
+}
+
+.search::placeholder {
+  font-family: Avenir, sans-serif;
+}
+
+.search:hover::placeholder {
+  font-family: Avenir, sans-serif;
+}
+
+.search:focus::placeholder {
+  color: transparent;
+}
+
+.search:focus {
+  background: url(images/Search.svg), linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-repeat: no-repeat;
+  outline: none;
+  box-sizing: border-box;
```

Comment:
box-sizing should be the same for all elements and not only on focused state, use * selector to set it for all elements